test/clients/firewall_auto_test.rb

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env rspec
+
+require_relative "../test_helper"
+require_relative "../../src/clients/firewall_auto.rb"
```

Comment:
btw nasty workaround for it is using `load File.expand_path("../../src/clients/firewall_auto.rb", __FILE__)` instead of calling main.